.circleci/doc_push.sh

```diff
@@ -12,7 +12,7 @@ pushd docs
 popd
 
 echo "Pushing to public"
-git config --global user.email "torchxla@gmail.com"
+git config --global user.email "pytorchxla@gmail.com"
```

Comment:
NB: why is this a gmail and not a corporate mail?

Reply:
^ it's the email used for creating a github account for CI integration. We used to share this with PyTorch folks (Meta).